resources/com.chatterino.chatterino2.appdata.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2019 Artem Polishchuk <ego.cordatus@gmail.com> -->
```

Comment:
Curious about best practices in terms of copyrighting contributions like this - generally any contribution would fall under the project license (MIT).
I'd prefer we just skip the copyright notice for now. I understand this makes our file not validate the strictest check, but that doesn't seem to be a real concern.

Reply:
No prob. I'll fix this shortly.